src/popup.html

```diff
@@ -9,6 +9,7 @@
         <hr/>
         <ul id="utils" class="toolsList">
             <li><a href="#" id="removeValidation">Disable HTML5 validation</a></li>
+            <li><a href="#" id="disableJavascript">Disable/Enable Javascript</a></li>
```

Comment:
Yep, I thought about that but didn't had time to test it, I'll take a look in the coming days to see if I can find something for this